nvi

```diff
@@ -91,6 +91,11 @@ ${B}SEE ALSO${N}
 EOF
 }
 
+fatal() {
+  printf 'Error: %s\n' "$1" 1>&2
```

Comment:
The reason why the redirection was prepended rather than appended is that the visual reconnaissance burden is (allegedly) lessened when you don't need to traverse some unknown length of characters down the line to find it. While skimming a script you might not always read to the end of each line, and particularly not if the line is a long string.

Admittedly this new line is so short as to make the issue academic, but how would you feel about prepending the redirection, anyway?